example/64_tensor_transforms/tensor_transform_using_wrapper.cpp

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#include <iostream>
+
+#include "ck/ck.hpp"
+
+#include "ck/utility/number.hpp"
+#include "ck/utility/tuple.hpp"
+#include "ck/utility/sequence.hpp"
+
+#include "tensor_transform_wrapper.hpp"
+
+using DataType = int;
+
+template <typename Layout>
+void Print(const Layout& layout)
+{
+    for(ck::index_t h = 0; h < ck::tensor_transform_wrapper::size<0>(layout); h++)
+    {
+        for(ck::index_t w = 0; w < ck::tensor_transform_wrapper::size<1>(layout); w++)
+        {
+            std::cout << layout(ck::make_tuple(h, w)) << " ";
+        }
+        std::cout << std::endl;
+    }
+}
+
+int main()
+{
+    // Basic descriptor 0, 1, 2, ... 30, 31 (runtime descriptor)
+    // (dims:4,8 strides:1,1)
+    const auto shape_4x8       = ck::make_tuple(4, 8);
+    const auto layout_4x8_s1x1 = ck::tensor_transform_wrapper::make_layout(shape_4x8);
+    std::cout << "dims:4,8 strides:1,1" << std::endl;
```

Comment:
Why the stride is 1 1 here, in cute, if stride is omitted, then strides will be derived as layout left and compact, in this case (1, 4)

Reply:
@cloudhan  Yes, You are right, fixed. Thanks :)